xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/ServerInfoIT.java

```diff
@@ -49,8 +50,8 @@ public void testServerInfoAcrossAllTypes() throws JsonRpcClientErrorException, I
     }
   }
 
-  private String getType(ServerInfo info) {
-    return info.map(
+  private String getType(ServerInfo result) {
```

Comment:
This rename seems a bit off. We should either leave it alone or rename it to `serverInfo`.